src/display/displayTemplateManager.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "displayCommon.h"
+
+namespace Templates {
+namespace Standard {
```

Comment:
I would like to put the namespaces into the respective headers instead of wrapping the `#include`.

Reply:
```cpp
namespace Templates::Standard {
    /* ... */
}
```

should work

Reply:
Namespace indentation you should get with 

```diff
diff --git a/.clang-format b/.clang-format
index 8dd84fa..a2b2426 100644
--- a/.clang-format
+++ b/.clang-format
@@ -32,6 +32,7 @@ IndentCaseLabels: true
 IndentCaseBlocks: true
 IndentRequires:  false
 IndentWidth:     4
+NamespaceIndentation: All
 PackConstructorInitializers: NextLineOnly
 PointerAlignment: Left
 QualifierAlignment: Leave
```

To apply this to all files in the repo, run

```
$ pre-commit run --all-files
```

otherwise the new format will only be applied to the changed files.

Reply:
The reason it didn't work here previously likely is that `#include` is no code but a pre-processor macro, and they are never indented.